### ISSUE TITLE
Error after starting compile.sh 

### ISSUE THREAD

**Issue by oneofthehorses** [Ifsttar/qrtone#27]
Hello! Thanks for your work!

I wish to figure out how to make program\site like yours (https://universite-gustave-eiffel.github.io/qrtone/) works locally without internet on my PC

Because no instructions i tryed this but fails... :

sh compile.sh 
emcc: warning: EXTRA_EXPORTED_RUNTIME_METHODS is deprecated, please use EXPORTED_RUNTIME_METHODS instead [-Wdeprecated] 

Would appreciate any help! :)

Maybe there any .html verison exists? :+1:

**Comment by nicolas-f** [comment]
Hello,

The message is just a warning so it will still produce the wasm file.

The website source code is here:

https://github.com/Universite-Gustave-Eiffel/qrtone/tree/gh-pages

**Comment by oneofthehorses** [comment]
https://github.com/Universite-Gustave-Eiffel/qrtone/tree/gh-pages

This helps a lot! Now works :)

BTW can i run it on android too? If so, how? Thankssss

**Comment by nicolas-f** [comment]
Yes it is already used in a Android Project. You just have to use the jqrtone library.

Here an Android project that use qrtone:

https://github.com/Universite-Gustave-Eiffel/NoiseCapture
